src/libs/utils.ts

```diff
@@ -72,8 +72,8 @@ const completeDateFormatParts = [
   new Intl.DateTimeFormat('default', { hour: 'numeric', minute: 'numeric' }),
 ];
 
-export const makePrettyDate = (dateString) => {
-  const date = new Date(dateString);
+export const makePrettyDate = (dateString: number | string | Date): string => {
```

Comment:
Alternatively, this type could be obtained from the Date type.

```suggestion
type DateConstructorArg = ConstructorParameters<typeof Date>;

export const makePrettyDate = (dateString: DateConstructorArg): string => {
```

Reply:
Hmm, the compiler/linter does not seem to like that one.